java/openmldb-taskmanager/src/main/java/com/_4paradigm/openmldb/taskmanager/config/TaskManagerConfig.java

```diff
@@ -94,8 +94,10 @@ public static void parse() throws IOException, NumberFormatException, ConfigExce
         ZK_MAX_CONNECT_WAIT_TIME = Integer.parseInt(prop.getProperty("zookeeper.max_connect_waitTime", "30000"));
 
         SPARK_MASTER = prop.getProperty("spark.master", "local").toLowerCase();
-        if (!Arrays.asList("local", "yarn", "yarn-cluster", "yarn-client").contains(SPARK_MASTER) ) {
-            throw new ConfigException("spark.master", "should be one of local, yarn, yarn-cluster or yarn-client");
+        if (!SPARK_MASTER.startsWith("local")) {
+            if (!Arrays.asList("yarn", "yarn-cluster", "yarn-client").contains(SPARK_MASTER)) {
+                throw new ConfigException("spark.master", "should be local, yarn, yarn-cluster or yarn-client");
+            }
         }
         boolean isLocal = SPARK_MASTER.equals("local");
```

Comment:
`isLocal` is false when master is `local[*]` ?

Reply:
Thanks and we should change this as well.